typescript/sdk/src/utils/env.ts

```diff
@@ -0,0 +1,11 @@
+import z from 'zod';
+
+const envScheme = z.object({
+  HYP_KEY: z.string().optional(),
+  ANVIL_IP_ADDR: z.string().optional(),
+  ANVIL_PORT: z.number().optional(),
+});
```

Comment:
Agreed– I've now removed the HYP_KEY from sdk-side, and similarly the ANVIL vars from the CLI-side.

Are you also against ANVIL vars sdk-side? We can certainly have these be passed into the fork util instead of parsed out of process.env

Reply:
^talking through it, I think that's my desired path now as well